addons/mail/static/src/core/common/plugin/mention_plugin.js

```diff
@@ -14,6 +23,27 @@ export class MentionPlugin extends Plugin {
         this.store = this.services["mail.store"];
     }
 
+    selectAll({ anchorNode, anchorOffset, focusNode, focusOffset }) {
+        // extend the selection to include whole mention elements at the borders so that it doesn't get stuck into the contenteditable=fasle
+        const SELECTOR = this.MENTION_SELECTORS.map(({ selector }) => selector).join(", ");
```

Comment:
Nitpick: comment above function is better to motivate whole function. Comment inside function is more appropriate for technical details or for commenting scope of if branch or loop.